sandbox/mock.go

```diff
@@ -41,7 +41,7 @@ func MockingSandbox(ts *testsuite.TestSuite) *MockSandbox {
 		TestCommittedTrxs:    make(map[tx.ID]*tx.Tx),
 	}
 
-	treasuryAmt := amount.Amount(21000000 * 1e9)
+	treasuryAmt := amount.Amount(21_000_000 * 1e9)
```

Comment:
can we have our decimals/conversion factor as a global parameter?

Reply:
This is only used in tests (note that it is not the total coins in network). Defining it globally for tests at this time is not a good idea unless we change the mocking structure for the sandbox. There is no good place for this global definition.
By the way, it is a good suggestion. We may work on this on another PR.

Reply:
No, not about coins amount. I'm talking about PAC coin decimals. I think we can define it globally.

Reply:
We try to work on that in other PRs.